src/ws/cockpitbranding.c

```diff
@@ -71,6 +72,17 @@ cockpit_branding_calculate_static_roots (const gchar *os_id,
       g_ptr_array_add (dirs, g_strdup_printf (DATADIR "/cockpit/branding/%s", os_id));
     }
 
+  if (os_id_like)
+    {
+      gchar **ids;
+
+      ids = g_strsplit_set (os_id_like, " ", -1);
+      for (gint i = 0; ids[i]; i += 1)
+        g_ptr_array_add (dirs, g_strdup_printf (DATADIR "/cockpit/branding/%s", ids[i]));
```

Comment:
Hmm, should we also add VARIANT_ID to each ID_LIKE here, like we do for ID?

Reply:
We discussed this, and we weren't sure.  So let's wait until it becomes clear what the right thing is.